util/db/repository.go

```diff
@@ -86,15 +86,15 @@ func (db *db) GetRepository(ctx context.Context, repoURL string) (*appsv1.Reposi
 		return nil, err
 	}
 
+	var repo = &appsv1.Repository{Repo: repoURL}
 	index := getRepositoryIndex(s, repoURL)
-	if index < 0 {
-		return nil, status.Errorf(codes.NotFound, "repo '%s' not found", repoURL)
-	}
-
-	repo, err := db.credentialsToRepository(s.Repositories[index])
-
-	if err != nil {
-		return nil, err
+	if index >= 0 {
+		repo, err = db.credentialsToRepository(s.Repositories[index])
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		log.Warn("repo not found, using dummy")
```

Comment:
Can we remove this log? It's going to spam warnings just because someone happens to use a public git repo.

Reply:
Ok.